static/js/hotkey.js

```diff
@@ -450,6 +450,11 @@ export function process_enter_key(e) {
         return true;
     }
 
+    // If we got this far and the modal is open, then avoid the enter press.
```

Comment:
// The logic below is intended for the main view; if we're in a modal, let the browser handle the key.